pkg/netutil/netutil_windows.go

```diff
@@ -0,0 +1,58 @@
+/*
+   Copyright The containerd Authors.
+
+   Licensed under the Apache License, Version 2.0 (the "License");
+   you may not use this file except in compliance with the License.
+   You may obtain a copy of the License at
+
+       http://www.apache.org/licenses/LICENSE-2.0
+
+   Unless required by applicable law or agreed to in writing, software
+   distributed under the License is distributed on an "AS IS" BASIS,
+   WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+   See the License for the specific language governing permissions and
+   limitations under the License.
+*/
+
+package netutil
+
+const (
+	DefaultNetworkName = "nat"
```

Comment:
Why is this different from Linux?

Reply:
There isn't a "bridge" network type for Windows AFAIK, and the Windows CNI plugins (at least the ones from https://github.com/microsoft/windows-container-networking: `nat`, `sdnoverlay`, and `sdnbridge`) require that the network name equals the network type: https://github.com/microsoft/windows-container-networking/issues/57

`sdnoverlay` and `sdnbridge` require extra system setup beyond the CNI config file, so they seem like poor choices for default usage. Third-party CNI plugins are equally likely to require extra system configuration.

Reply:
The network name is expected to match Docker's default network, not the name of CNI plugin.

Reply:
Ah, I misunderstood. I thought this was what ended up substituted as as `.Name` in `ConfigListTemplate`.

Reply:
When using Docker the default network on Windows is called `nat` so I called it that here. Are you saying this should stay `bridge`?

Reply:
nat SGTM then